doc/src/variable.txt

```diff
@@ -670,6 +670,16 @@ sequence of output timesteps:
 
 100,150,200,...950,1000,1500,2000,...9500,10000,15000,etc :pre
 
+The logfreq3(x,y,z) function generates y points between x and z (inclusive),
+that are separated by a multiplicative ratio: (z/x)^(1/(y-1)). Constraints
+are: x,z > 0, y > 1, z-x >= y-1. For eg., if logfreq3(10,25,1000) is used in
+a variable by the "fix print"_fix_print.html command, then the interval
+between 10 and 1000 is divided into 24 parts with a multiplicative
+separation of ~1.21, and it will generate the following sequence of output
+timesteps:
+
+10, 13, 15, 18, 22, 27, 32,...384, 465, 563, 682, 826, 1000 :pre
+
```

Comment:
I want to make sure that the code implements the requirement metioned for logfreq(),
namely
For
any current timestep, the next timestep in the sequence is returned.
That is required for it to used robustly in input scripts where the
current timestep could be anything.

Reply:
Yes, it'd taken me a bit to understand that I had to generate the next time-step, and not the current. I think I got it right in the end. I did also test various scenarios.